### PR TITLE
About link has just been  added

### PR DESCRIPTION
**Pull request by Batmanwilly2022**
I have just added About link 
kindly for futher

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,6 +14,7 @@ <h1>GitHUb Branch, Merge</h1>
         <ul>
             <li>Home</li>
             <li>Contact</li>
+            <li>About</li>
```
Comment (EmmaKbl) [inline index.html]: change About to Services

**Review by EmmaKbl — CHANGES_REQUESTED** [review]
@Batmanwilly2022 Have just added a comment in the code to let you know the changes to do.

File: index.html (hunk 1 above)
Comment (Yunusib12) [reply index.html]: @IdrissaMurenga could you take this and work on it as @Batmanwilly2022 is out today?

Comment (IdrissMulenga) [reply index.html]: @Yunusib12 All fixed

**Review by EmmaKbl — APPROVED** [review]

**Comment by EmmaKbl** [comment]
Looks great!

**Pull request merged**